windows/win_iis_webapppool.ps1

```diff
@@ -90,10 +90,14 @@ try {
       Stop-WebAppPool -Name $name -ErrorAction Stop
       $result.changed = $TRUE
     }
-    if ((($state -eq 'started') -and ($pool.State -eq 'Stopped')) -or ($state -eq 'restarted')) {
+    if ((($state -eq 'started') -and ($pool.State -eq 'Stopped'))) {
       Start-WebAppPool -Name $name -ErrorAction Stop
       $result.changed = $TRUE
     }
+    if ($state -eq 'restarted') {
+      Restart-WebAppPool -Name $name
```

Comment:
I understand what you're going for with restarted, I'm just saying that as implemented, the restarted state fails when the pool is not already running (because restart-apppool doesn't start a stopped one). You need to account for that case somehow.